cmd/collection/main.go

```diff
@@ -139,13 +141,25 @@ func main() {
 				"additional fraction of replica timeout that the primary will wait for votes")
 			flags.DurationVar(&blockRateDelay, "block-rate-delay", 250*time.Millisecond,
 				"the delay to broadcast block proposal in order to control block production rate")
+			flags.StringVar(&startupTimeString, "startup-time", cmd.NotSet,
```

Comment:
Did you consider using unix timestamp ? I feel like it's less error phone in regards of timezones.

Reply:
yeah, I like @durkmurder 's idea of using timestamp. 

The example `2006-01-02T15:04:05Z07:00` here is still confusing to me. Like the `Z07:00` part. 

A timestamp like `1632245321894` is easier.

Reply:
It's better to be specific that only the hotstuff is stopped until the startup time
```suggestion
			flags.StringVar(&startupTimeString, "hotstuff-startup-time", cmd.NotSet,
```